app/controllers/NextUpdatesController.scala

```diff
@@ -75,21 +74,15 @@ class NextUpdatesController @Inject()(NoNextUpdatesView: NoNextUpdates,
             Future.successful(errorHandler.showInternalServerError())
           case (_, true) =>
             auditNextUpdates(user, isAgent, origin)
-
-            val optOutSetup = {
-              for {
-                checks <- optOutService.getNextUpdatesQuarterlyReportingContentChecks
-                optOutOneYearViewModel <- optOutService.nextUpdatesPageOptOutViewModel()
-                _ <- optOutService.resetSavedIntent()
-              } yield Ok(nextUpdatesOptOutView(viewModel, optOutOneYearViewModel, checks, backUrl.url, isAgent, origin))
-            }.recoverWith {
+            (for {
+              nextUpdatesOptOutViewModel <- optOutService.getNextUpdatesOptOutViewModel()
```

Comment:
not covered by unit test
![Screenshot 2024-07-08 at 16 07 07](https://github.com/hmrc/income-tax-view-change-frontend/assets/165788042/fce88da1-78a9-4227-8eac-b83b3500c875)